submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/common/BaseIntegrationTest.kt

```diff
@@ -16,4 +25,30 @@ internal open class BaseIntegrationTest(private val tempFolder: TemporaryFolder)
         System.setProperty("app.tempDirPath", temp.absolutePath)
         System.setProperty("app.security.filesDirPath", dropbox.absolutePath)
     }
+
+    protected fun getWebClient(serverPort: Int, user: TestUser): BioWebClient {
```

Comment:
IMO this method make 100% sense for this generic base class but if you check others as `submitString` they does not bellow to all test, they do help to reuse code but there is no a reason to all integration test have that method available.

Reply:
Done